frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -19,13 +22,42 @@ const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefini
     cniVersion: '0.3.1',
   };
 
-  _.forOwn(typeParamsData, (val, key) => {
-    config[key] = _.get(val, 'value', null);
-  });
+  let ipam = {};
+  try {
+    ipam = JSON.parse(_.get(typeParamsData, 'ipam.value', {}));
+  } catch (e) {
+    console.log(e); // eslint-disable-line no-console
+  }
+
+  if (networkType === 'cnv-bridge') {
+    const vlan = _.get(typeParamsData, 'vlanTagNum.value', '');
+    config.plugins = [
+      {
+        type: 'cnv-bridge',
```

Comment:
repeating string, worth of a constant

Reply:
Agreed. This will be addressed in a follow-up.